nixos/modules/services/misc/rmfakecloud.nix

```diff
@@ -22,6 +20,14 @@ in {
         '';
       };
 
+      serviceDataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/rmfakecloud";
+        description = lib.mdDoc ''
+          Directory to store rmfakecloud data at.
```

Comment:
```suggestion
          The rmfakecloud directory used to store all data. If left as the default value
          this directory will automatically be created before the rmfakecloud server starts, otherwise
          you are responsible for ensuring the directory exists with appropriate ownership
          and permissions.
```